src/main/kotlin/com/criptext/mail/db/models/Email.kt

```diff
@@ -97,6 +97,8 @@ data class Email(
                                 json.getString("trashDate"), null) else null
 
                 )
+                email.headers = if(json.has("headers")) json.getString("headers") else null
```

Comment:
Because is something that we don't need to persist in the db, only in the file system. Having it this way, it will be ignored by the database and does not need to be passed a parameter each time we need to instantiate one of this objects. This filed is only needed when syncing devices, since in the json the headers field will be present, and needs to be stored somewhere temporarily until is written in the file system.